Jenkinsfile

```diff
@@ -228,7 +228,7 @@ pipeline {
                 {
                     agent{ label rocmnode("gfx908")}
                     environment{
-                        setup_args = """ -D CMAKE_CXX_FLAGS="--offload-arch=gfx908 -O3 " -DBUILD_DEV=On """
+                        setup_args = """ -D CMAKE_CXX_FLAGS=" --offload-arch=gfx900 --offload-arch=gfx906  --offload-arch=gfx908 --offload-arch=gfx90a -O3 " -DBUILD_DEV=On """
```

Comment:
Maybe the stage name `"Run Tests: gfx908"` should be changed if you build for multiple architectures?

Reply:
All architectures are only being built, only `gfx908` is being tested since only one type of node is assigned to the CI stage